src/test/java/org/osiam/client/AbstractIntegrationTestBase.java

```diff
@@ -50,8 +50,8 @@
     protected static final Client CLIENT = ClientBuilder.newBuilder().register(JacksonFeature.class).build();
 
     static {
-        OsiamConnector.setConnectTimeout(10000);
-        OsiamConnector.setReadTimeout(30000);
+        OsiamConnector.setConnectTimeout(Integer.parseInt(System.getProperty("debug.timeout.connect", "-1")));
+        OsiamConnector.setReadTimeout(Integer.parseInt(System.getProperty("debug.timeout.read", "-1")));
```

Comment:
The reason for setting the default values to `-1` here is that the defaults will only be active when the tests are started from the IDE. I think we can assume that this means, one wants to debug. And we can not know about the invoked maven profile from within the IDE. I think this is the most pragmatic solution for this problem.